packages/e2e-tests/specs/__snapshots__/adding-blocks.test.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`adding blocks Should insert content using the placeholder and the regul
 <!-- /wp:quote -->
 
 <!-- wp:image -->
-<figure class=\\"wp-block-image\\"><img alt=\\"\\"/></figure>
+<figure class=\\"wp-block-image\\"><img src=\\"\\" alt=\\"\\"/></figure>
```

Comment:
Why did this change?

Reply:
because that's what --updateSnapshot decided :) will check!

Reply:
I think this is correct as by default the save function will add a src attribute.

Reply:
Well I love this. Now the snapshot is wrong! :)